bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/thing/modely.xml

```diff
@@ -119,6 +120,7 @@
 			<channel id="valetmode" typeId="valetmode"/>
 			<channel id="valetpin" typeId="valetpin"/>
 			<channel id="wakeup" typeId="wakeup"/>
+			<channel id="wiperbladeheater" typeId="wiperbladeheater"/>
```

Comment:
This is an undocumented change. 
Maybe you can create a separate PR for that?

Reply:
Where should I document this? Instead of separate PR, rather let's document it wherever applicable.

Reply:
In the description of your PR. 
But as it is something unrelated to the object of this PR, a separate PR would have been clearly better in particular for the release notes and to inform users. 
If you prefer a unique PR with multiple objects, at least describe all if them.

Reply:
OK. I got your point. It makese sense to have it as a separate PR so I did it. Please approve the other one first and then we can go over this one. I think I've fixed your comments (mostly).

Reply:
I believe you fully misunderstood my comment. 
My comment was not about steeringwheelheater but about wiperbladeheater you also added. Is there an obvious link between them? 

For simply steeringwheelheater, there is of course no need to create several PRs.